modules/file-input/test/spec.coffee

```diff
@@ -16,116 +18,265 @@ describe "File Input", ->
     lastModified: new Date()
     type: 'image/png'
 
+  fakeEvent = {
+    # fake some dom event stuff
+    stopPropagation: ->
+  }
 
-  beforeEach ->
+  before ->
     hx.consoleWarning = chai.spy()
-    fixture.clear()
+    hx._.dropdown.attachToSelector = '#fixture'
+    clock = sinon.useFakeTimers()
```

Comment:
Came across a random issue where 2 of the data table event tests were failing.
Defining the spies / clock etc. in the before function seemed sensible anyway - probably worth doing this everywhere?